### ISSUE TITLE
Regression tests in this GT4Py use non-installed headers from GridTools

### ISSUE THREAD

**Issue by havogt** [GridTools/gt4py#497]
Related issue: the regression tests in this repository use non-installed headers from GridTools. When I install GridTools using this nix derivation https://github.com/VulcanClimateModeling/packages/tree/master/gridtools/v1.nix, and setting `GT_INCLUDE_DIR`, I get the following error from pytest:
```
/Users/johannd/Code/gt4py/tests/reference_cpp_regression/reference.cpp:17:10: fatal error: 'gridtools/regression/horizontal_diffusion_repository.hpp' file not found
#include "gridtools/regression/horizontal_diffusion_repository.hpp"
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
```

For now I will change the derivations to manually copy over the regression directory to the installation path, but it's a workaround.

_Originally posted by @jdahm in https://github.com/GridTools/gt4py/issues/457#issuecomment-893760664_

**Comment by jdahm** [comment]
If the `pip` installed package works, then this should be done. See https://github.com/GridTools/gridtools/pull/1720.

**Comment by jdahm** [comment]
Solved in #1032.
